07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,59 @@
+#top{
+  background-image: url("../img/nasa-background.jpg");
+  text-align: center;
+  width: 850px;
+  height: 450px;
+  margin-bottom: 15px;
+}
+
+#top>h1{
+  color: white;
+  font-family: "Arial Black", sans-serif;
+  font-weight: bold;
```

Comment:
give this h1 a margin of 0 so that it doesn't create distance from the top of the page.